pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a 10 columns wide display:
```

Comment:
On another note, I can't seem to get this option to produce any different behavior regardless of what number I use. Any hints?